Formula/or-tools.rb

```diff
@@ -0,0 +1,66 @@
+class OrTools < Formula
+  desc "Google's Operations Research tools"
+  homepage "https://developers.google.com/optimization/"
+  license "Apache-2.0"
+
+  url "https://github.com/google/or-tools/archive/v8.0.tar.gz"
+  sha256 "ac01d7ebde157daaeb0e21ce54923a48e4f1d21faebd0b08a54979f150f909ee"
+
+  head "https://github.com/google/or-tools/archive/master.zip"
+
+  bottle do
+    cellar :any
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "abseil"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "protobuf"
+  depends_on "cbc"
+  depends_on "cgl"
+  depends_on "clp"
+  depends_on "osi"
+  depends_on "coinutils"
+
+  def install
+    # CMake based build
+    system "cmake", "-S.", "-Bbuild", *std_cmake_args, "-DUSE_SCIP=OFF", "-DBUILD_SAMPLES=OFF", "-DBUILD_EXAMPLES=OFF"
+    system "cmake", "--build", "build", "-v"
+    system "cmake", "--build", "build", "--target", "install"
+
+    # Produce pkg-config file under cmake/make
```

Comment:
This should be submitted as a patch upstream, we don't make our own pkg-config files.

Reply:
ok removed